.github/workflows/bot-nonreg-nitrogen.yml

```diff
@@ -34,3 +33,4 @@ jobs:
           SLACK_ICON_EMOJI: ":bot-seed7:"
           SEED: ${{ secrets.SEED7 }}
           BOT_ENVIRONMENT: production
+          BOT_DISABLED_CURRENCIES: bitcoin,ethereum,qtum,decred,cardano,axelar,cosmos,secret_network,avalanche_c_chain,bsc,filecoin,tron,cronos,fantom,boba,telos_evm,polygon_zk_evm,polkadot
```

Comment:
Why is this needed ?

Reply:
I think this is linked to (cf. description of the PR):

> Disabled on non reg : Bitcoin, Ethereum as it costs too much and we test on testnets
>Weekly basis: qtum,decred,cardano,axelar,cosmos,secret_network,avalanche_c_chain,bsc,filecoin,tron,cronos,fantom,boba,telos_evm,polygon_zk_evm,polkadot
>Daily basis : the rest

So basically:
- Bitcoin and Ethereum are always disabled on non reg (weekly or daily)
- On the weekly non reg, only `qtum,decred,cardano,axelar,cosmos,secret_network,avalanche_c_chain,bsc,filecoin,tron,cronos,fantom,boba,telos_evm,polygon_zk_evm,polkadot` run
- On a daily basis, everything else runs (i.e, everything that does not run on a weekly basis, also excluding bitcoin and ethereum that never run on non reg)

Reply:
Could be worth a comment directly in the workflow files for better understanding (present and future), for example:

For `bot-nonreg-oxygen`:
```
# We only run these currencies on a weekly basis because (...)
BOT_FILTER_CURRENCIES: (...)
```

For `bot-nonreg-nitrogen`:
```
# We don't run these currencies daily because (...)
BOT_DISABLED_CURRENCIES: (...)
```